hail/python/hail/experimental/import_gtf.py

```diff
@@ -160,3 +162,86 @@ def import_gtf(path, reference_genome=None, skip_invalid_contigs=False, min_part
     ht = ht.key_by('interval')
 
     return ht
+
+
+@typecheck(gene_symbols=nullable(sequenceof(str)),
+           gene_ids=nullable(sequenceof(str)),
+           transcript_ids=nullable(sequenceof(str)),
+           verbose=bool, reference_genome=nullable(reference_genome_type), gtf_file=nullable(str))
+def get_gene_intervals(gene_symbols=None, gene_ids=None, transcript_ids=None,
+                       verbose=True, reference_genome=None, gtf_file=None):
+    """Get intervals of genes or transcripts.
+
+       Get the boundaries of genes or transcripts from a GTF file, for quick filtering of a Table or MatrixTable.
+
+       Gencode v19 (GRCh37) GTF available at: gs://hail-common/references/gencode/gencode.v19.annotation.gtf.bgz
+       Gencode v29 (GRCh38) GTF available at: gs://hail-common/references/gencode/gencode.v29.annotation.gtf.bgz
+
+       Example
+       -------
+       >>> hl.filter_intervals(ht, get_gene_intervals(gene_symbols=['PCSK9']))  # doctest: +SKIP
```

Comment:
mind adding a `reference_genome='GRCh37'` in the example? People make a lot of mistakes related to this.